context/app/static/js/components/entity-tile/EntityTileTop/style.js

```diff
@@ -25,8 +24,24 @@ const StyledDonorIcon = styled(DonorIcon)`
 // Width needs to be defined in px for text-overflow to work
 const FixedWidthFlex = styled.div`
   display: flex;
-  width: ${tileWidth};
   padding: 10px 10px;
+
+  background-color: #ffffff;
+
+  svg {
+    color: ${(props) => props.theme.palette.primary.main};
+  }
+
+  ${(props) =>
+    props.$invertColors &&
+    css`
+      background-color: ${props.theme.palette.primary.main};
+      color: #ffffff;
+
+      svg {
+        color: #ffffff;
+      }
+    `}
```

Comment:
`EntityTileBottom/style.js` also has this: Can you factor it out to a place they could share from?

Reply:
Similar but the colors are swapped.